server/src/main/java/io/unitycatalog/server/service/credential/CredentialContext.java

```diff
@@ -0,0 +1,30 @@
+package io.unitycatalog.server.service.credential;
+
+import java.net.URI;
+import java.util.List;
+import java.util.Set;
+import lombok.Builder;
+import lombok.Getter;
+
+@Builder
+@Getter
+public class CredentialContext {
+  public enum Privilege {
+    SELECT,
+    UPDATE
+  }
+
+  private String storageScheme;
+  private String storageBase;
+  private Set<Privilege> privileges;
+  private List<String> locations;
```

Comment:
Is this defined as a list to be future proof since the constructor currently only sets one location corresponding to the `locationURI`?

Reply:
correct

Reply:
Can you add a comment to explain since the rest of the code works with just a single string?